sanitize.go

```diff
@@ -0,0 +1,57 @@
+package scroll
+
+// The AllowSet interface is implemented to detect if input is safe or not.
+type AllowSet interface {
+	IsSafe(string) bool
+}
+
+// AllowSetChars allows the definition of a set of safe allowed characters.
+type AllowSetChars struct {
```

Comment:
I would prefer `AllowSetBytes` to make it more clear that this does not know about unicode.